inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/utils/WorkerUtils.kt

```diff
@@ -0,0 +1,24 @@
+package com.rakuten.tech.mobile.inappmessaging.runtime.utils
+
+import androidx.work.ListenableWorker
+import timber.log.Timber
+
+internal object WorkerUtils {
+    fun reportError(tag: String, code: Int, message: String?) {
+        Timber.tag(tag).e("Response Code $code: ${message ?: "null message"}")
+    }
+
+    fun reportSilentError(tag: String, code: Int, message: String?) {
```

Comment:
Is that method used for reporting request related error only? If so, then the name doesn't reflect that (same for `reportError()`

Reply:
updated.
please check.